.github/workflows/copy-workflow.yml

```diff
@@ -33,6 +33,12 @@ jobs:
         if [[ ! -f .github/workflows/go-test.yml ]]; then
           echo "INITIAL_TEST_DEPLOYMENT=1" >> $GITHUB_ENV
         fi
+    - name: remove the autorebase workflow
```

Comment:
perhaps add a TODO to remove this eventually? perhaps in a month's time, if we think all files are gone in master by then.